Donation-platform/src/main/java/com/google/servlets/TagScoreUpdateServlet.java

```diff
@@ -25,11 +25,13 @@
 import javax.servlet.http.HttpServletResponse;
 import com.google.model.Charity;
 import com.google.model.Tag;
+import com.google.DbCalls;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Enumeration;
+import java.io.*;
```

Comment:
I've heard it's bad practice to use the wildcard. https://medium.com/@aayushsharda/why-is-it-better-to-avoid-in-import-statements-in-java-ff9ceaca8aeb
Maybe consider importing libraries needed only.

Reply:
fixed